tests/test_helper.py

```diff
@@ -57,3 +59,53 @@ def test_with_expected_and_args_and_kwargs(self):
         expect(split_expected_params_str).to.contain("'a'")
         expect(split_expected_params_str).to.contain("b='b'")
         expect(split_expected_params_str).to.contain('one=1')
+
+
+class TestUnicodeStringToStr(TestCase):
+    def test_unicode_string_to_str(self):
+        self.assertEqual(Helper.unicode_string_to_str(u'Mèo'), u'Mèo'.encode('utf-8'))
+
+
+class TestUnicodeListToStrList(TestCase):
+    @patch('tests.test_helper.Helper.unicode_to_str')
```

Comment:
I just want to know if the right function calls the right things. Moreover, because Python 2 and 3 deal with unicode strings differently, asserting for real return values here is rather complicated.